### PR TITLE
Update commands to install NodeJS and NPM.

### PR DESCRIPTION
**Pull request by dannywillems**
I always use NVM because I know there is some issues with the version in the package manager on Debian/Ubuntu.

### CODE REVIEW

File: src/mobile.wiki [hunk 1]
```diff
@@ -63,10 +63,16 @@ software.
 
 For all mobile platforms:
 
-* Install {{{npm}}} (on Debian and Ubuntu, {{{sudo apt-get install npm}}}).
+* Install {{{NodeJS}}} and {{{NPM}}} (see
+  [[https://nodejs.org/en/download/package-manager/|the official website]]). If
+  you have some issues with your package manager, you can also use
+  [[https://github.com/creationix/nvm|nvm]].
```
Comment (vasilisp) [inline src/mobile.wiki]: The official website encourages bad practices like piping `curl $RANDOM_URL` to `sudo bash` and relies on repositories of dubious quality. The Node ecosystem we know and love.

So I much prefer providing a shortcut for the most common setup (Debian/Ubuntu), and letting users of the other distributions and the *BSDs adapt. They are smart enough :).

Comment (dannywillems) [inline src/mobile.wiki]: @vasilisp

> The Node ecosystem we know and love.

:laughing:

Are you sure NPM suffices? NodeJS must be installed no? I only used once the package manager for Node and I switched to NVM.

File: src/mobile.wiki [hunk 2]
```diff
@@ -63,10 +63,16 @@ software.
...
 * Install Cordova ({{{sudo npm install -g cordova}}})
 * Install the needed Cordova plugins:\\
-  {{{sudo npm install -g cordova-hot-code-push-cli read-package-json xml2js}}}
+  {{{sudo npm install -g cordova-hot-code-push-cli read-package-json xml2js}}}.
```
Comment (dannywillems) [inline src/mobile.wiki]: @vasilisp Are you sure `read-package-json` and `xml2js` are mandatory?

File: src/mobile.wiki (hunk 1 above)
Comment (vasilisp) [inline src/mobile.wiki]: NodeJS will get installed as a dependency of NVM.

**Comment by dannywillems** [comment]
Oups...

File: src/mobile.wiki (hunk 1 above)
Comment (dannywillems) [inline src/mobile.wiki]: @vasilisp Ok, I will change to the initial version for the installation.

File: src/mobile.wiki (hunk 2 above)
Comment (dannywillems) [inline src/mobile.wiki]: I think we can let them. It's safer...

**Comment by dannywillems** [comment]
Old version. Not up-to-date. I close.

**Pull request closed without merging**